.github/workflows/lint.yaml

```diff
@@ -0,0 +1,70 @@
+name: Lint
+
+on:
+  pull_request:
```

Comment:
I see that you created a new `.yaml` file to run the linters and validators. 

Question: is it possible to add the two jobs from this file into `continuous-integration.yaml`? That way we have all the jobs centralized.